### PR TITLE
Sass grid

### PR DESCRIPTION
**Pull request by rosscreighton**
per our text discussion

### CODE REVIEW

File: imports/ui/stylesheets/index.scss [hunk 1]
```diff
@@ -0,0 +1,4 @@
+@import "{wolves:bourbon}/bourbon";
```
Comment (sameckmeier) [inline imports/ui/stylesheets/index.scss]: we could even have a file called libraries and stash bourbon and neat in it and any other libraries we end up using

File: imports/ui/stylesheets/index.scss [hunk 2]
```diff
@@ -0,0 +1,4 @@
...
+@import "{wolves:neat}/neat";
+
+@import "./components/index.scss";
```
Comment (sameckmeier) [inline imports/ui/stylesheets/index.scss]: i like this, and seems like the best patch over until, as we said, a more developed meteor implementation of css modules comes out. i dont think this work flow is gonna be that much of a headache. i like how modularized this is. plus, this feels very meteor-y 👍

**Comment by sameckmeier** [comment]
looks good to me and seems to be the best workflow for until a more developed css modules comes out

File: imports/ui/stylesheets/index.scss (hunk 1 above)
Comment (rosscreighton) [inline imports/ui/stylesheets/index.scss]: yeah good idea. idk if we'll end up having enough css libraries to justify a separate file, but if we do should do that.

File: imports/ui/stylesheets/index.scss (hunk 2 above)
Comment (rosscreighton) [inline imports/ui/stylesheets/index.scss]: yep, agreed

**Pull request merged**